config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -241,6 +241,10 @@ presubmits:
               value: "true"
             - name: USE_CI_ARTIFACTS
               value: "true"
+            - name: FEATURE_GATE_MACHINE_POOL
+              value: "true"
+            - name: USE_MACHINE_POOL
```

Comment:
would it make sense to use the same var for both? Or maybe set `USE_MACHINE_POOL` to the `${FEATURE_GATE_MACHINE_POOL}`? I can't think of a situation where you'd want to set one to true but not the other

Reply:
I will consolidate them into `FEATURE_GATE_MACHINE_POOL`.